src/workspaces/NewWorkspaceModal/NewWorkspaceModal.test.ts

```diff
@@ -989,54 +582,10 @@ describe('NewWorkspaceModal', () => {
     });
   });
 
-  it('emits a metrics event when creating an Azure workspace', async () => {
-    // Arrange
-    const user = userEvent.setup();
-
-    const billingProjectWithRegion = _.cloneDeep(azureBillingProject);
-    billingProjectWithRegion.region = 'eastus';
-
-    const { createWorkspace, captureEvent } = setup({ billingProjects: [billingProjectWithRegion] });
-    createWorkspace.mockResolvedValue(defaultAzureWorkspace.workspace);
-
-    await act(async () => {
-      render(
-        h(NewWorkspaceModal, {
-          onSuccess: () => {},
-          onDismiss: () => {},
-        })
-      );
-    });
-
-    // Act
-    const workspaceNameInput = screen.getByLabelText('Workspace name *');
-    act(() => {
-      fireEvent.change(workspaceNameInput, { target: { value: 'Test workspace' } });
-    });
-
-    const projectSelect = new SelectHelper(screen.getByLabelText('Billing project *'), user);
-    await projectSelect.selectOption(/Azure Billing Project/);
-
-    const createWorkspaceButton = screen.getByRole('button', { name: 'Create Workspace' });
-    await user.click(createWorkspaceButton);
-
-    // Assert
-    expect(createWorkspace).toHaveBeenCalled();
-    const expectedEvent = {
-      cloudPlatform: 'AZURE',
-      region: 'eastus',
-      workspaceName: defaultAzureWorkspace.workspace.name,
-      workspaceNamespace: defaultAzureWorkspace.workspace.namespace,
-      hasProtectedData: undefined,
-      workspaceAccessLevel: undefined,
-    };
-    expect(captureEvent).toHaveBeenCalledWith(Events.workspaceCreate, expectedEvent);
-  });
-
-  it.each([
-    { billingProjectName: azureBillingProject.projectName, cloudPlatform: 'Azure' },
-    { billingProjectName: gcpBillingProject.projectName, cloudPlatform: 'Gcp' },
-  ] as { billingProjectName: string; cloudPlatform: WorkspaceInfo['cloudPlatform'] }[])(
+  it.each([{ billingProjectName: gcpBillingProject.projectName, cloudPlatform: 'Gcp' }] as {
```

Comment:
Again, don't need a `.each` here - there might be other spots as well where it can be made into a single test instead of an iteration over one input.

Reply:
Fixed: https://github.com/DataBiosphere/terra-ui/pull/5275/commits/35cd2b7baf08f54330812a1e0bcd1236f2d67d90